packages/apollo-link-schema/src/schemaLink.ts

```diff
@@ -40,7 +40,7 @@ export class SchemaLink extends ApolloLink {
           this.schema,
           operation.query,
           this.rootValue,
-          this.context,
+          this.context || operation.getContext(),
```

Comment:
I'm not entirely sure about this. Would a merge be better?

Reply:
Why do you need to pass in `context` to the SchemaLink constructor? I think using just `operation.getContext()` makes sense here.

Reply:
Yeah, that makes sense and I can remove the constructor param which is the original API. At first, I didn’t want to remove it completely, breaking dependent apps without approval.